### PR TITLE
feat: remove sudo

### PR DESCRIPTION
**Pull request by martinfridrich**
Sudo pallet was removed from basilisk runtime.

some root only actions shifted to the council and technical comitee

### CODE REVIEW

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]
we should shift some configuration tasks to the technical comitee majority

**Comment by martinfridrich** [comment]
> we should shift some configuration tasks to the technical comitee majority

Which one? Most of the pallets are already configured `...TechCommitteeOrRoot`.

**Comment by lumir-mrkva** [comment]
> > we should shift some configuration tasks to the technical comitee majority
> 
> Which one? Most of the pallets are already configured `...TechCommitteeOrRoot`.

ok, i'll review

**Review by lumir-mrkva — APPROVED** [review]

File: runtime/basilisk/src/xcm.rs [hunk 1]
```diff
@@ -136,15 +136,15 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type ChannelInfo = ParachainSystem;
 	type VersionWrapper = PolkadotXcm;
-	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
-	type ControllerOrigin = crate::EnsureMajorityCouncilOrRoot;
+	type ExecuteOverweightOrigin = MajorityTechCommitteeOrRoot;
+	type ControllerOrigin = MajorityTechCommitteeOrRoot;
```
[the remark this answers is not in the document]
Comment (lumir-mrkva) [reply runtime/basilisk/src/xcm.rs:140]: yes, these roles are more for sorting out technical issues

File: runtime/basilisk/src/lib.rs [hunk 2]
```diff
@@ -737,8 +717,7 @@ impl pallet_tips::Config for Runtime {
 impl pallet_collator_selection::Config for Runtime {
 	type Event = Event;
 	type Currency = Balances;
-	//allow 1/2 of council to execute privileged collator selection operations. (require code from: feat/initial_chain_setup)
-	type UpdateOrigin = EnsureMajorityCouncilOrRoot;
+	type UpdateOrigin = MajorityTechCommitteeOrRoot;
```
[the remark this answers is not in the document]
Comment (lumir-mrkva) [reply runtime/basilisk/src/lib.rs:720]: yes, currently required to maintain stable running chain

**Pull request merged**